### PR TITLE
WnS Due Validation on PT title transfer (Contribution from Impel team)

### PR DESCRIPTION
**Pull request by rakesh-wt-egov**

### CODE REVIEW

File: web/rainmaker/dev-packages/egov-ui-kit-dev/src/common/propertyTax/Property/components/PendingAmountDue/index.js [hunk 1]
```diff
@@ -31,28 +48,66 @@ class PendingAmountDialog extends Component {
...
-              <Button
-                label={<Label buttonLabel={true} label="PT_PROCEED_TO_PAY" fontSize="16px" labelClassName="footer-button-label" />}
-                primary={true}
-                className="footer-button"
-                onClick={() => { this.navigateToCommonPay(envURL) }}
-              />
+            <div>
+              <Label fontSize="18px" label="PT_PROPERTY_DUE" />
+              <div style={payNowButton} >
+                <div style={labelStyle}>Rs {amount}</div>
```
Comment (jagankumar-egov) [inline web/rainmaker/dev-packages/egov-ui-kit-dev/src/common/propertyTax/Property/components/PendingAmountDue/index.js:66]: Use localisation key for Rs

similar to 
     <div className="text-right" style={{ marginTop: 10 }}>
              <Label className="footer-amount" labelClassName="footer-amount" label="PT_MUTATION_RS" fontSize="24px" /><span className="footer-amount-no" > {amount}</span>

File: web/rainmaker/dev-packages/pgr-employee-dev/src/Screens/CreateComplaint/components/AddComplaintForm/index.js [hunk 2]
```diff
@@ -23,12 +23,14 @@ const AddComplaintForm = ({
   const { name, phone, mohalla, city, address, landmark, houseNo } = fields;
   const submit = form.submit;
   if(city && city.dropDownData && city.dropDownData.length > 0) {
+    debugger;
```
Comment (jagankumar-egov) [inline web/rainmaker/dev-packages/pgr-employee-dev/src/Screens/CreateComplaint/components/AddComplaintForm/index.js:26]: remove these debuggers;

File: web/rainmaker/dev-packages/egov-ui-kit-dev/src/common/propertyTax/Property/components/PendingAmountDue/index.js (hunk 1 above)
Comment (rakesh-wt-egov) [reply web/rainmaker/dev-packages/egov-ui-kit-dev/src/common/propertyTax/Property/components/PendingAmountDue/index.js]: Added localization key for Rs

File: web/rainmaker/dev-packages/pgr-employee-dev/src/Screens/CreateComplaint/components/AddComplaintForm/index.js (hunk 2 above)
Comment (rakesh-wt-egov) [reply web/rainmaker/dev-packages/pgr-employee-dev/src/Screens/CreateComplaint/components/AddComplaintForm/index.js]: removed debuggers

**Review by jagankumar-egov — APPROVED** [review]

**Pull request merged**